src/bc.F

```diff
@@ -1585,7 +1585,6 @@ subroutine radbcns4(rvf,radbcs,radbcn,v1,v2,dt)
             radbcs(i,k)=avgs
           enddo
         enddo
```

Comment:
does't make sense to me. Why do we have to remove `!$acc end parallel`?

Reply:
@sjsprecious, there is no corresponding '$acc parallel' here, but rather a '$acc parallel loop'.  Apparently, nvhpc is not bothered by this but CCE is.

Reply:
It should be ok to match `!$acc parallel loop` with `!$acc end parallel` since there is no `!$acc end loop`. I guess this issue is caused by the fact that CCE is supporting an old OpenACC standard?

Reply:
I don't know if it is standard issues or the nvhpc compiler that is more forgiving.  Regardless, I am hoping to eventually get the OpenACC version of CM1 to compile using both NVHPC and CCE compilers.